release_images.yml

```diff
@@ -2,9 +2,25 @@
 release_images:
   1:
     framework: "tensorflow"
-    version: "2.2.0"
-    inference:
+    version: "1.15.3"
+    training:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py37"]
+      python_versions: ["py37", "py36"]
       os_version: "ubuntu18.04"
-      cuda_version: "cu102"
+      cuda_version: "cu100"
+      example: False        # [Default: False] Set to True to denote that this image is an Example image
+      disable_sm_tag: True  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
```

Comment:
Why are we disabling sm_tag? Checked pro repo for tf1.15.2 and it is enabled

Reply:
Sorry, I have updated it.